<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Auto">
       <attribute name="title">
        <string>Auto</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Car ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
       <widget class="QLabel" name="id">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="name">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>60</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="sname">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="l4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tonn</string>
        </property>
       </widget>
       <widget class="QLabel" name="l4_o">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>140</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="l5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Price per tonn</string>
        </property>
       </widget>
       <widget class="QLabel" name="l5_o">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>180</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Truck">
       <attribute name="title">
        <string>Truck</string>
       </attribute>
       <widget class="QLabel" name="l4_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tonn</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Car ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="l5_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Price per tonn</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="id_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="sname_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="l4_o_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>140</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="name_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>60</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
       <widget class="QLabel" name="l5_o_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>180</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Bus">
       <attribute name="title">
        <string>Bus</string>
       </attribute>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Car ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="l5_o_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>180</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="sname_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>100</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="l4_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tonn</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
       <widget class="QLabel" name="l4_o_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>140</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="name_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>60</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="l5_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Price per tonn</string>
        </property>
       </widget>
       <widget class="QLabel" name="id_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="car">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="truck">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Get info</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="bus">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="winner">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>378</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
